### PR TITLE
feat: add project and environment columns to events

### PR DESCRIPTION
**Pull request by chriswk**

### CODE REVIEW

File: src/lib/services/event-service.ts [hunk 1]
```diff
@@ -28,6 +28,10 @@ export default class EventService {
             (e: IEvent) => e.type !== FEATURE_METADATA_UPDATED,
         );
     }
+
+    async getEventsForProject(project: string): Promise<IEvent[]> {
+        return this.eventStore.getEventsFilterByProject(project);
```
Comment (ivarconr) [inline src/lib/services/event-service.ts:33]: **side note:*'
All stores now support `getAll(query: QT)` so we should in theory be able to combine these into a single getAll-call to the store.

**Review by ivarconr — APPROVED** [review]

File: src/lib/types/events.ts [hunk 2]
```diff
@@ -9,6 +9,9 @@ export const FEATURE_REVIVED = 'feature-revived';
 export const FEATURE_IMPORT = 'feature-import';
 export const FEATURE_TAGGED = 'feature-tagged';
 export const FEATURE_TAG_IMPORT = 'feature-tag-import';
+export const FEATURE_STRATEGY_UPDATE = 'feature-strategy-update';
+export const FEATURE_STRATEGY_ADD = 'feature-strategy-add';
+export const FEATURE_STRATEGY_REMOVE = 'feature-strategy-remove';
```
Comment (ivarconr) [inline src/lib/types/events.ts:14]: These new events needs to be available to all addons as well. Any idea on how to also apply them to existing addon config where the user may only subscribe to FEATURE_UPDATE?

File: src/lib/services/event-service.ts (hunk 1 above)
Comment (chriswk) [reply src/lib/services/event-service.ts:33]: ooh, good point. Let's make a separate task for that :)

File: src/lib/types/events.ts (hunk 2 above)
Comment (chriswk) [reply src/lib/types/events.ts:14]: Should I make the service send both events for now, and we'll revisit this if/when we discuss events again?

Comment (ivarconr) [reply src/lib/types/events.ts:14]: yes

**Review by FredrikOseberg — APPROVED** [review]
LGTM

**Pull request merged**